dependency/dependency.go

```diff
@@ -38,7 +38,7 @@ type List struct {
 type Info struct {
 	Name                    string `json:"name"`
 	Dir                     string `json:"-"`
-	LicenceFile             string `json:"-"`
+	LicenceFile             string `json:"licenceFile"`
```

Comment:
Wouldn't this require providing the full path to the licence file in the module cache directory?  The reason for disallowing it was for two reasons:
- Portability: to prevent OS-specific or user-specific paths from breaking the override rules
- Security: to prevent someone from accidentally (or intentionally) outputting contents of sensitive files 

The workaround for the problem is to define the `licenceTextOverrideFile` which is handled more securely by chrooting to the project directory (example: https://github.com/elastic/cloud-on-k8s/blob/master/hack/licence-detector/overrides/overrides.json). Will that not work for your use case too?

Reply:
Ah, I didn't realise that the relative path would work because my memory was that it required the full path. That takes away my main concern about portability (although it might still break on Windows but nobody develops on Windows right? :smile: ) 

+1 on using the same `securepath` methods on the licence file path to ensure that  the file is only within the go module cache.